src/novo/src/client.tsx

```diff
@@ -0,0 +1,26 @@
+import "desktop/lib/webpackPublicPath"
+
+import React from "react"
+import ReactDOM from "react-dom"
+import { buildClientApp } from "v2/Artsy/Router/client"
+import { getAppNovoRoutes } from "v2/Apps/getAppNovoRoutes"
+import { data as sd } from "sharify"
+import { loadableReady } from "@loadable/component"
+
+import { mediator } from "lib/mediator"
+
+buildClientApp({
+  routes: getAppNovoRoutes(),
```

Comment:
Args should be just `routes`, see: https://github.com/artsy/force/blob/f431126f291c602c9f14354d16f2a333348fc28b/src/desktop/apps/artsy-v2/client.tsx#L12

`mediator` and `user` are now required by default here: https://github.com/artsy/force/blob/c1752c4c53d7d37424f4bea75194fb577678ec48/src/desktop/lib/buildClientAppContext.ts#L12:14

Reply:
Relevant PRs for changes are:
https://github.com/artsy/force/pull/6482
https://github.com/artsy/force/pull/6511